academy/4-my-own-chain/game-winner.md

```diff
@@ -26,7 +26,7 @@ In this section, you will:
 
 </HighlightBox>
 
-To be able to terminate games you need to identify games that have already been terminated. A good field to add is for the **winner**. It needs to contain:
+To be able to terminate games you need to identify games that have already been played. A good field to add is for the **winner**. It needs to contain:
```

Comment:
```suggestion
To be able to terminate games you need to identify games that have already been terminated. A good field to add is for the **winner**. It needs to contain:
```
It really is "terminated" because you don't want to terminate a game a second time.

Reply:
WIP

Reply:
No no, don't deploy this sentence with "played" instead of "terminated".